backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultProjectService.kt

```diff
@@ -14,6 +15,9 @@ class DefaultProjectService(
     private val conversionService: ConversionService,
 ) : ProjectService {
 
+    override suspend fun get(id: String) = projectComponent.get(ObjectId(id))
```

Comment:
same

and check the other places, please

Reply:
fixed